rpi-source

```diff
@@ -77,11 +80,11 @@ def debug(str):
 
 def info(str):
     if not args.quiet:
-        print("\n *** %s" % str)
+        print(("\n *** %s" % str))
```

Comment:
I'm fairly sure the extra brackets you've added here and on line 87 aren't necessary?